vm/src/vm/context/run_context.rs

```diff
@@ -10,9 +10,9 @@ use crate::{
 use num_traits::abs;
 
 pub struct RunContext {
-    pub(crate) pc: Relocatable,
-    pub(crate) ap: usize,
-    pub(crate) fp: usize,
+    pub pc: Relocatable,
+    pub ap: usize,
+    pub fp: usize,
```

Comment:
Please keep these private. If you need to, you may implement getters that return either the offset as `usize` or the corresponding `Relocatable`. I think we have those already tho.

Reply:
Fixed